### DISCUSSION TITLE
The last four days.

### DISCUSSION THREAD

**Discussion by arif48** [IBAX-io/go-ibax#1173] [Soru Cevap - Türkçe]
Will it be successful among other networks? What are your thoughts?

**Comment by Ammymond1** [comment]
On Thu, 27 Jan 2022 at 23:54 arifpecen ***@***.***> wrote:

> Will it be successful among other networks? What are your thoughts?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IBAX-io/go-ibax/discussions/1173>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXP5OBXDY6AVUNY6A75INRDUYHEKHANCNFSM5M7H5AAA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Sure, this is a super project,successful is 100% sure--
Best Regards
Amarachi
Director
Global Education Overseas Studies Limited

Head Office:
Gate Way Plaza,4th Floor
Zakariya Maimalari Road
Harbert Macaulay Way, CBD
Abuja,Nigeria

Tel:  +234 8137164580
               8110170977

Whatsapp: +234 8137164580
Email: ***@***.***

**Reply by Yusufknk** [reply]
Bunu ilerleyen zamanlarda ki gelişmeler gösterecektir.
